acceptance/allocator_terraform/load.tf

```diff
@@ -4,6 +4,10 @@ variable "example_block_writer_instances" {
   default = 1
 }
 
+variable "load_duration" {
+  default = "0" # unlimited
```

Comment:
`block_writer` needs to terminate and print benchmark stats as I made it do in https://github.com/cockroachdb/examples-go/pull/72 (currently blocked because of `grpc-go`-related build issue).

I guess I could make the stats print in a `defer` block in `block_writer` so that it always shows up.